spec/requests/create_payment_for_successful_request_spec.rb

```diff
@@ -0,0 +1,98 @@
+require 'spec_helper'
+
+describe "Create payments on successful charge" do
+  let(:event_data) do
+    {
+    "id" => "evt_15h8i8KoSw2hrlQeVpYKLNgG",
+    "created" => 1426552652,
+    "livemode" => false,
+    "type" => "charge.succeeded",
+    "data" => {
+      "object" => {
+        "id" => "ch_15h8i8KoSw2hrlQeq9MZaYBZ",
+        "object" => "charge",
+        "created" => 1426552652,
+        "livemode" => false,
+        "paid" => true,
+        "status" => "succeeded",
+        "amount" => 999,
+        "currency" => "usd",
+        "refunded" => false,
+        "source" => {
+          "id" => "card_15h8i7KoSw2hrlQeKuWBU76i",
+          "object" => "card",
+          "last4" => "4242",
+          "brand" => "Visa",
+          "funding" => "credit",
+          "exp_month" => 10,
+          "exp_year" => 2015,
+          "fingerprint" => "yVufS9dNGxll2kqT",
+          "country" => "US",
+          "name" => nil,
+          "address_line1" => nil,
+          "address_line2" => nil,
+          "address_city" => nil,
+          "address_state" => nil,
+          "address_zip" => nil,
+          "address_country" => nil,
+          "cvc_check" => "pass",
+          "address_line1_check" => nil,
+          "address_zip_check" => nil,
+          "dynamic_last4" => nil,
+          "metadata" => {},
+          "customer" => "cus_5suXMRv9oPbV7R"
+        },
+        "captured" => true,
+        "balance_transaction" => "txn_15h8i8KoSw2hrlQen2Y9sxqD",
+        "failure_message" => nil,
+        "failure_code" => nil,
+        "amount_refunded" => 0,
+        "customer" => "cus_5suXMRv9oPbV7R",
+        "invoice" => "in_15h8i8KoSw2hrlQeyAviaYmR",
+        "description" => nil,
+        "dispute" => nil,
+        "metadata" => {},
+        "statement_descriptor" => "Myflix fee",
+        "fraud_details" => {},
+        "receipt_email" => nil,
+        "receipt_number" => nil,
+        "shipping" => nil,
+        "refunds" => {
+          "object" => "list",
+          "total_count" => 0,
+          "has_more" => false,
+          "url" => "/v1/charges/ch_15h8i8KoSw2hrlQeq9MZaYBZ/refunds",
+          "data" => []
+        }
+      }
+    },
+    "object" => "event",
+    "pending_webhooks" => 1,
+    "request" => "iar_5suXcAzpmsjSEx",
+    "api_version" => "2015-02-18"
+    }
```

Comment:
I was thinking about this code here and whether I liked it or not - but I have come to a conclusion in my head I don't mind as it makes the test easy to read and understand what is happening here.